tools/sof_perf_analyzer.py

```diff
@@ -234,7 +237,9 @@ def analyze_perf_info():
     perf_stats.columns = ['CPU_AVG(MIN)', 'CPU_AVG(AVG)', 'CPU_AVG(MAX)',
                           'CPU_PEAK(MIN)', 'CPU_PEAK(AVG)', 'CPU_PEAK(MAX)']
     perf_stats['PEAK(MAX)/AVG(AVG)'] = perf_stats['CPU_PEAK(MAX)'] / perf_stats['CPU_AVG(AVG)']
-
+    perf_stats['MODULE_CPC'] = perf_stats['CPU_PEAK(AVG)'] * perf_info['PERIOD'][1] * CPC_MARGIN
```

Comment:
This doesn't look good, `perf_info['PERIOD'][1]` is a single value for one component, it is used by all component in this code. 

the assignment to `perf_stats['MODULE_CPC']` could be done in a single line of code.  The code here will degrade the performance.

`astype` will do `floor` to the data, this means 1.9 becomes 1, make sure this is the expected behavior.

Reply:
> This doesn't look good, `perf_info['PERIOD'][1]` is a single value for one component, it is used by all component in this code.
any good ideas for better implementation?
> 
> the assignment to `perf_stats['MODULE_CPC']` could be done in a single line of code. The code here will degrade the performance.
could show some sample code? I don't know how to implement.
> 
> `astype` will do `floor` to the data, this means 1.9 becomes 1, make sure this is the expected behavior.
yes, however, this is cycle level truncation, cycle level impact is very minor.